qoqo/src/noise_models/continuous_decoherence.rs

```diff
@@ -77,14 +77,47 @@ impl ContinuousDecoherenceModelWrapper {
         }
     }
 
-    /// Return the internal Lindblad noise operator of the continuous noise model.
+    /// Return the internal Lindblad noise operator of the ContinuousDecoherenceModel noise model.
     ///
     /// Returns:
-    ///     PlusMinusLindbladNoiseOperator: The internal Lindblad noise operator of the continuous noise
-    pub fn get_noise_operator(&self) -> PlusMinusLindbladNoiseOperatorWrapper {
-        PlusMinusLindbladNoiseOperatorWrapper {
-            internal: struqture::spins::PlusMinusLindbladNoiseOperator::from(self.internal.clone()),
-        }
+    ///     PlusMinusLindbladNoiseOperator: The internal Lindblad noise operator of the ContinuousDecoherenceModel.
+    pub fn get_noise_operator(&self) -> Py<PyAny> {
+        Python::with_gil(|py| {
+            let binding = py
+                .import_bound("importlib.metadata")
+                .expect("Could not import importlib.metadata module for get_noise_operator")
+                .getattr("version")
+                .expect("Could not get version function of importlib.metadata")
+                .call1(("struqture_py",))
+                .expect("Could not get version attribute of struqture_py");
+            let version: &str = binding.extract().expect("Could not extract version string");
+            if version.starts_with('1') {
+                let class = py
+                    .import_bound("struqture_py.spins")
+                    .expect("Could not import struqture_py.spins module for get_noise_operator")
+                    .getattr("PlusMinusLindbladNoiseOperator")
+                    .expect("Could not get PlusMinusLindbladOperator class");
```

Comment:
We might want to put getting the struqture version and extracting the class in the module setup code in lib.rs and put the results in a OnceCell so we don't have to redo the import every time for speed reasons.